protocol/signalfx/spanobfuscation/spantagremoval_test.go

```diff
@@ -137,7 +138,7 @@ func BenchmarkRmTen(b *testing.B) {
 	config := make([]*TagMatchRuleConfig, 0, 10)
 	for i := 0; i < 9; i++ {
 		rule := &TagMatchRuleConfig{
-			Service:   pointer.String("some*test*service" + string(i)),
+			Service:   pointer.String("some*test*service" + strconv.Itoa(i)),
```

Comment:
can we split this fix in separate commit

Reply:
Done in #23